src/kayenta/canary.less

```diff
@@ -43,8 +43,7 @@
     margin-bottom: 40px;
     overflow-y: auto;
     .config-detail-header {
-      padding-right: 15px;
-      padding-bottom: 7px;
+      padding: 5px 15px 7px 0;
```

Comment:
the "JSON" button retains focus after the modal closes, but there's no padding in the containing div so the border shadow only shows on 3 sides; adding some padding so it looks a little less bad